egs/swbd/s5c/local/chain/run_tdnn_6z.sh

```diff
@@ -55,7 +55,7 @@ final_layer_normalize_target=0.5
 num_jobs_initial=3
 num_jobs_final=16
 minibatch_size=128
-relu_dim=576
+relu_dim=1024
```

Comment:
Do you mean the final affine transform represented by "0" at the end of the splice indexes? Actually, if you look at swbd/s5c/local/nnet3/run_tdnn.sh, you will see this 0 at the end of the splice indexes, as well [here](https://github.com/kaldi-asr/kaldi/blob/master/egs/swbd/s5c/local/nnet3/run_tdnn.sh#L57). Even though my understanding is that the point of the final affine component is for using cross-entropy training as a regularizer in chain models, this has snuck into the cross entropy model as well. (I haven't done a git blame to find out when this originally appeared.) Since the splice indexes are exactly the same, I believe relu_dim should be the same between the the cross entropy model and chain model in order to guarantee that they have the same number of parameters, as they will use the same neural network when we set the splice indexes the same. At the least, I convinced my self the relu_dim should be the same between the two models to guarantee the same number of parameters with an equation I came up with for number of parameters as a function of relu_dim.

Also, sorry about the delay. I know we need to get this out ASAP.